### PR TITLE
Internal review - Pool, CompoundingIndex, VariableUtilisationRatesCalculator

### PR DESCRIPTION
**Pull request by piotruda**

### CODE REVIEW

File: contracts/CompoundingIndex.sol [hunk 1]
```diff
@@ -103,6 +103,9 @@ contract CompoundingIndex is Ownable {
     }
 
 
+    /**
+      * returns compounded factor in Ray
```
Comment (kamilovsky01) [inline contracts/CompoundingIndex.sol:107]: **R**eturns? :)

File: contracts/Pool.sol [hunk 2]
```diff
@@ -61,7 +61,7 @@ error UnauthorizedBorrower();
 error BorrowFromEmptyPool();
 
 /// The pool utilisation cannot be greater than 95%
-error PoolUtilisationTooHigh();
+error PoolUtilisationTooHighForBorrow();
```
Comment (kamilovsky01) [inline contracts/Pool.sol:64]: I'd say PoolUtilisationTooHighToBorrow or PoolUtilisationTooHighForBorrowing. What do you think?

File: contracts/Pool.sol [hunk 3]
```diff
@@ -73,7 +73,7 @@ error PoolUtilisationTooHigh();
  */
 contract Pool is OwnableUpgradeable, ReentrancyGuardUpgradeable, IERC20 {
 
-  uint256 public constant MAX_POOL_UTILISATION = 0.95 ether;
+  uint256 public constant MAX_POOL_UTILISATION_FOR_BORROW = 0.95 ether;
```
Comment (kamilovsky01) [inline contracts/Pool.sol:76]: Same as above

File: contracts/Pool.sol [hunk 4]
```diff
@@ -230,6 +230,7 @@ contract Pool is OwnableUpgradeable, ReentrancyGuardUpgradeable, IERC20 {
   **/
   function borrow(uint256 _amount) payable external canBorrow nonReentrant {
     require(address(this).balance >= _amount);
+    //max pool utilisation for borrowing is checked in canBorrow modifier
```
Comment (kamilovsky01) [inline contracts/Pool.sol:233]: // **M**ax pool [...]

File: contracts/Pool.sol [hunk 5]
```diff
@@ -385,7 +386,7 @@ contract Pool is OwnableUpgradeable, ReentrancyGuardUpgradeable, IERC20 {
     if (!_borrowersRegistry.canBorrow(msg.sender)) revert UnauthorizedBorrower();
     if (totalSupply() == 0) revert BorrowFromEmptyPool();
     _;
-    if (totalBorrowed() * 1 ether / totalSupply() > MAX_POOL_UTILISATION) revert PoolUtilisationTooHigh();
+    if (totalBorrowed() * 1 ether / totalSupply() > MAX_POOL_UTILISATION_FOR_BORROW) revert PoolUtilisationTooHighForBorrow();
```
Comment (kamilovsky01) [inline contracts/Pool.sol:389]: Same as the first comment

File: contracts/Pool.sol (hunk 2 above)
Comment (piotruda) [reply contracts/Pool.sol:64]: For Borrowing

**Review by kamilovsky01 — APPROVED** [review]

**Pull request merged**